config/setup/yurt-tunnel-agent.yaml

```diff
@@ -78,6 +79,10 @@ spec:
           valueFrom:
             fieldRef:
               fieldPath: status.podIP
+        - name: NODE_IP
+          valueFrom:
+            fieldRef:
+              fieldPath: status.hostIP
```

Comment:
The value of POD_IP is same as node ip because hostNetwork field is true. so we should keep only one env here.

Reply:
Ah, I didn't notice that. Will remove the NODE_IP as we will always use the host network for the tunnel component.

Reply:
ok, keep the NODE_IP env and remove the POD_IP env, but keep hostNetwork is true.